packages/js/client-config-builder/src/bundles/getDefaultConfig.ts

```diff
@@ -34,6 +34,7 @@ export const defaultWrappers = {
   graphNode: "wrap://ens/wraps.eth:graph-node@1.0.0",
   ensTextRecordResolver:
     "wrap://ipfs/QmbqeVAhSzTtSmdVjrPMK42pX1sFs8t5MUB741T7nxSs1p",
+  githubResolver: "wrap://ipfs/QmTNukpDR6HHtZnUKX88ytJTfqRaeLoy6bpDLRXAQX42sW",
```

Comment:
This should probably be done within another PR

Reply:
Yes, for sure. This is a mistake. I initially added it but then I thought I removed it in this commit https://github.com/polywrap/toolchain/pull/1592/commits/97df57a58df49d4a12f7d2700094842305f174e7 . But it looks like I only removed it from the uri resolver ext implementations array

Reply:
fixed!